cmd/client/main.go

```diff
@@ -166,6 +166,7 @@ func (c *Coordinator) doPush(resp *http.Response, origRequest *http.Request, cli
 	url := base.ResolveReference(u)
 
 	buf := &bytes.Buffer{}
+	//nolint:errcheck
 	resp.Write(buf)
```

Comment:
Be explicit vs disabling with commentary 🤗 
```suggestion
	_, _ = resp.Write(buf)
```